libs/platform/src/lib/components/form/form-group/form-field/form-field.component.ts

```diff
@@ -97,6 +93,9 @@ export class FormFieldComponent implements FormField, AfterContentInit, AfterVie
     @Input()
     i18Strings: TemplateRef<any>;
 
+    @Input()
```

Comment:
same comment for the /** */

Reply:
https://github.com/SAP/fundamental-ngx/issues/4247